app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/ChannelLayoutManager.kt

```diff
@@ -37,7 +61,12 @@ class ChannelLayoutManager(
         startSmoothScroll(smoothScroller)
     }
 
-    class FirstSmoothScroller(context: Context) : LinearSmoothScroller(context) {
+    /**
+     * [FirstSmoothScroller] is designed to support carousel scrolling and scrolling speed
+     */
+    class FirstSmoothScroller(private val context: Context) : LinearSmoothScroller(context) {
+
+        // Scrolling speed
         override fun calculateSpeedPerPixel(displayMetrics: DisplayMetrics?): Float {
```

Comment:
Should we add a comment where this calculation came from?

Reply:
It's an overriden method but I can add a general comment

Reply:
Maybe just saying what needed to be changed? Could not be needed, your call.